sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ChangeFeedProcessorTest.java

```diff
@@ -424,6 +427,46 @@ public void readFeedDocumentsAfterSplit() {
         receivedDocuments.clear();
     }
 
+    @Test(groups = { "simple" }, timeOut = CHANGE_FEED_PROCESSOR_TIMEOUT)
+    public void testServiceItemLeaseSerialization() {
```

Comment:
thank you for the test. as a code style rule we don't add "test" as prefix.

Please drop the "test" prefix to be consistent with other tests.

Reply:
done